srcopsmetrics/storage.py

```diff
@@ -98,37 +141,36 @@ def get_ceph_store(self) -> CephStore:
         s3.connect()
         return s3
 
-    def load_previous_knowledge(self, project_name: str, file_path: Path, knowledge_type: str) -> Dict[str, Any]:
+    def load_previous_knowledge(
+        self, project_name: str = None, knowledge_type: str = None, file_path: Optional[Path] = None
+    ) -> Dict[str, Any]:
         """Load previously collected repo knowledge. If a repo was not inspected before, create its directory.
 
         Arguments:
-            repo_path {Path} -- path of the inspected github repository
+            file_ath {Path} -- path of the inspected github repository
```

Comment:
`file_path`

Reply:
not sure what you meant
is it correct now?

Reply:
I read `file_ath` :D

Reply:
oh damn!